requirements/base.in

```diff
@@ -18,4 +18,5 @@ psycopg2
 python-qpid-proton
 requests
 requests-gssapi
+https://github.com/spdx/tools-python/archive/fd6aeba22ec0ad7650571d99db19d2a113ed0782.zip
```

Comment:
Is there a reaso we're downloading a particular Github commit instead of a PyPI release?